docs/ja/api-reference/ngsild.md

```
@@ -324,7 +343,9 @@ DELETE /ngsi-ld/v1/entities/{entityId}/attrs/{attrName}
 | パラメータ | 型 | 説明 |
 |-----------|-----|------|
 | `datasetId` | string | 削除するマルチ属性インスタンスの datasetId |
-| `deleteAll` | boolean | `true` の場合、すべてのインスタンスを削除します |**レスポンス**: `204 No Content`### マルチ属性 (datasetId)
+| `deleteAll` | boolean | `true` の場合、すべてのインスタンスを削除します |**レスポンス**: `204 No Content`
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)